scaleway/helpers_object.go

```diff
@@ -16,6 +20,9 @@ func newS3Client(region, accessKey, secretKey string) (*s3.S3, error) {
 	config.WithRegion(region)
 	config.WithCredentials(credentials.NewStaticCredentials(accessKey, secretKey, ""))
 	config.WithEndpoint("https://s3." + region + ".scw.cloud")
+	if os.Getenv("S3_DEBUG") != "" {
```

Comment:
should we rather use `SCW_DEBUG` ?

Reply:
It seems more user friendly yes, even if they are two different SDKs

Reply:
We went with `TF_LOG==debug`